wordle-web/components/SettingsDialog.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <v-dialog v-model="modelValue" max-width="500">
+    <v-card>
+      <v-sheet color="primary">
+        <v-card-text> Settings </v-card-text>
+      </v-sheet>
+      <v-card-text>
+        Dark/Light:   
+        <v-btn 
+          icon="mdi-theme-light-dark" @click="toggleTheme" />
+      </v-card-text>
+      
+      <v-sheet class="ml-6">
+        <v-select
+          color="secondary"
+          v-model="chosenTheme"
+          label="Choose a Theme"
+          :items="['Standard', 'Mystic Meadow', 'Enchanted Eclipse']"
+        ></v-select>
+        <v-btn color="secondary" @click="changeTheme" class="mb-6" type="submit">Activate</v-btn>
+      </v-sheet>
```

Comment:
```suggestion
        <v-btn color="secondary" @click="changeTheme" class="mb-6" text="Activate" />
```
There isn't a reason to have the `type=submit` on the button because we aren't submitting forms. This button can also have a self closing tag since it doesn't contain any content that can't be defined within its default props.

Reply:
I used an example from the docs to create this so that is why I had those in there.  Thank you for explaining, helped me understand the v-btn a bit better.